compose/release-notes.md

```diff
@@ -7,6 +7,16 @@ redirect_from:
   - /release-notes/docker-compose/
 ---
 
+## 1.25.5
+(2020-04-10)
+
+### Features
+
+- Bump OpenSSL from 1.1.1d to 1.1.1f
+
+- Add 3.8 compose version
+Limit service scale to the size specified by the field "deploy.placement.max_replicas_per_node"
```

Comment:
Missing the list item format `-`.
Limited service scale to the size specified by the field `deploy.placement.max_replicas_per_node`.

Reply:
Yep. Actually I would like to put this under the "- Add 3.8 compose version" like a sub-item

Reply:
- Added Compose version 3.8.

  - Limited service scale to the size specified by the field `deploy.placement.max_replicas_per_node`.

Reply:
This adds a sub-item under the Compose version entry.

Reply:
Done!